header.php

```diff
@@ -3,14 +3,15 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title><?php echo $title ?></title>
+    <title>Welcome to Hämeenlinna</title>
```

Comment:
Why did you delete this echo title? When for example in login page, now it's Welcome to Hämeenlinna? Deepak wants to see the title to change on different pages like he taught us, so this method $title will give us more points rather than Welcome to Hämeenlinna everywhere.